src/App.tsx

```diff
@@ -1,32 +1,30 @@
-import React, { useState, useEffect } from 'react';
-import axios from 'axios';
+import React from "react";
 
-import { RegistrationForm } from "./components/Registration";
+import { ThemeProvider } from "styled-components";
+import { RegistrationForm } from "./components";
 import Footer from "components/Footer";
```

Comment:
ten footer powinno dać się zaimportować w sposób
import {Footer} from 'componetns'

Reply:
Footer to testowy komponent, który zrobiła Weronika. Pewnie będzie do usunięcia.